From 0141a3ed4d49e964bcfffa3d73024b71dc81a244 Mon Sep 17 00:00:00 2001
From: Kalesh Singh <kaleshsingh@google.com>
Date: Mon, 6 Dec 2021 16:45:59 -0800
Subject: [PATCH] Allow gpuservice to read tracefs entries

Gpu service needs tracefs access to attach the BPF program
to the gpu_mem_total tracepoint for GPU memory tracking.

Bug: 208892266
Bug: 209513178
Test: adb shell dumpsys gpu --gpumem
Change-Id: I10fd54544bee940313f3987247c1841101a9d03a
---
 services/gpuservice/gpuservice.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/gpuservice/gpuservice.rc b/services/gpuservice/gpuservice.rc
index 65a5c2776a..0da8bd3812 100644
--- a/services/gpuservice/gpuservice.rc
+++ b/services/gpuservice/gpuservice.rc
@@ -1,4 +1,4 @@
 service gpu /system/bin/gpuservice
     class core
     user gpu_service
-    group graphics
+    group graphics readtracefs
-- 
2.39.0

